src/kibana/plugins/dashboard/styles/main.less

```diff
@@ -72,8 +72,9 @@ dashboard-grid {
 
         .panel-title {
           font-size: inherit;
+          display: inline-block;
```

Comment:
Ok. I was testing in ie 11, which seems to have fixed it. Please leave that line in.